### ISSUE TITLE
[Regression] WWE '13 BLES01699 now constantly freezing and hangs in recent RPCS3 updates. 

### ISSUE THREAD

**Issue by Bobble80** [RPCS3/rpcs3#16108]
### Quick summary

There has been a performance regression in the WWE '12 BLUS30841 and WWE '13 BLES01699 games in recent updates to RPCS3. The games still boot, but will freeze or hang during initial loading or when they get to the title screen.

### Details

I usually test WWE '13 every couple of months in RPCS3 to see if the performance is improving, and it does seem to be getting better as time goes on. A few months ago, and even beginning from July 2023 when I initially tested the game for the first time, I could always get ingame and play some matches. Maybe after 20 mins or so, the game might freeze and hang but it was playable for the most part. 

Now I've been testing the game over the last week or two and there seems to be a big hit to the performance in recent updates. I don't know exactly when this happened or in what build of RPCS3, but now in the current builds the game will boot as normal but will freeze and hang either during initial loading or when it gets to the title screen.
I've also tested the previous game in the series, WWE '12, and getting the same issue.

### Build with regression

Unknown

### Attach two log files

[RPCS3.log.gz](https://github.com/user-attachments/files/17084052/RPCS3.log.gz)

**Comment by AniLeo** [comment]
Need Information: Build that introduced the regression.

**Comment by Bobble80** [comment]
> Need Information: Build that introduced the regression.

I've just tested it from build rpcs3-v0.0.32-16650-f948a802_win64 from June 30th and it seems to be running ok like before with no freezing or hangs.

Do I have to check every build now from June 30th onwards to find out where the regression was?

**Comment by Darkhost1999** [comment]
> Do I have to check every build now from June 30th onwards to find out where the regression was? 

Yes, you do gotta bisect the issue down to 1 build that introduced the issue to narrow down the cause.
